setup.py

```diff
@@ -205,6 +204,42 @@ def lz4_extension():
 
     return extensions
 
+def zfp_extension():
+    import numpy
+    info('setting up ZFP extension')
+
+    extra_compile_args = list(base_compile_args)
+    define_macros = []
+
+    # setup sources - use ZFP bundled in blosc
+    zfp_sources = glob('c-blosc/internal-complibs/zfp*/src/*.c')
+    include_dirs = [d for d in glob('c-blosc/internal-complibs/zfp*/*') if os.path.isdir(d)]
```

Comment:
I'm confused about this. zfp is not part of blosc. Why are you looking for it there?